### PR TITLE
es6-const-let | Danny Andreev

### PR DESCRIPTION
**Pull request by dannyandreev**
image

### CODE REVIEW

File: es6-const-let/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>es6-const-let</title>
+</head>
+<body>
+  <script>
+    var greetings = [
```
Comment (scbowler) [inline es6-const-let/index.html:11]: This shouldn't be `var`

**Review by scbowler — CHANGES_REQUESTED** [review]
1 update needed

File: es6-const-let/index.html (hunk 1 above)
Comment (dannyandreev) [reply es6-const-let/index.html]: fixed!

**Review by scbowler — APPROVED** [review]
Well done! 👍

**Pull request merged**